scripts/kubectl-workspace-ensure

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose: ensure that a workload management workspace exists and has
+# the needed APIBindings.
+
+# Usage: kubectl ws ensure name [--enter] [-V|--verbose]
+
+set -e
+
+get_cws() {
+    echo "$(kubectl ws . --short 2> /dev/null)"
+}
+
+echoerr() {
+    >&2 echo "$*"
+}
+
+enter="false"
+verbose="false"
+cws=$(get_cws)
+ws=""
+
+while (( $# > 0 )); do
+    case "$1" in
+	(--enter)
+	    enter="true";;
+	(-h|--help)
+	    echo "Usage: kubectl ws ensure name [--enter] [-h|--help] [-V|--verbose] [-X]"
+	    exit 0;;
+	(-V|--verbose)
+	    verbose="true";;
+	(-X)
+	    set -x;;
+	(-*)
```

Comment:
FYI: since this script invokes `kubectl`, ideally it would be able to pass all the global command line flags that kubectl takes (arguably excepting `--context` because of the way `kubectl ws` uses the context setting).